src/components/Home/Products/Products.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react'
+import axios from "axios"
+import Gallery from "../Carousel/Carousel"
+import Logo from "../../../../static/images/logo.png"
+import "./Products.scss"
+import {withRouter} from "react-router-dom"
+ function Products(props) {
+    const [products, setProducts] = useState([])
+
+
+    useEffect(() => {
+        const apiCall = async () => {
+            let result = await axios.get(`http://localhost:5000/categories`)
```

Comment:
again, don't use axios here, create common function in utils folder and use it everywhere